### DISCUSSION TITLE
Does it work with Next.js 14 and Route Handlers?

### DISCUSSION THREAD

**Discussion by michele-gobbi** [jellydn/next-swagger-doc#947] [General]
I'm creating an API using Next.js 14.1.0 using the [route handlers](https://nextjs.org/docs/app/building-your-application/routing/route-handlers) with the new App Router. 
Does next-swagger-doc still work?
Thanks

**Comment by jellydn** [comment]
Do you check the examples app? There is a next 14 demo on this repo. Let me know if any issues

**Reply by michele-gobbi** [reply]
Oh thank you! I had missed the examples!
